itou/www/stats/utils.py

```diff
@@ -61,6 +61,13 @@ def can_view_stats_siae_etp(request):
     )
 
 
+def can_view_stats_siae_orga_etp(request):
```

Comment:
Il y a déjà une fonction `can_view_stats_siae_etp` qui semble assez connexe.
J'imagine que la logique de ce nouveau setting est de whitelister plutôt des organisation plutôt que des utilisateurs ?
Est-ce que ça a vocation à remplacer le setting STATS_SIAE_USER_PK_WHITELIST ou venir en complément ?

La fonction `can_view_stats_siae_etp` vérifie que l'utilisateur est dans la whitelist et qu'il est admin de l'organisation.
Est-ce qu'il faut faire la même chose avec ce nouveau setting ? 
Ou est-ce que tous les utilisateurs de l'organisation whitelistée peuvent voir les stats ETP ?


return (
        can_view_stats_siae(request)
        and request.is_current_organization_admin
        and request.user.pk in settings.STATS_SIAE_USER_PK_WHITELIST
)

Reply:
En gros on a deux cas de figure
- Un premier ou l'admin d'une structure peut voir les stats etp de toutes les structures dont il est membre et la on utilise bien STATS_SIAE_USER_PK_WHITELIST et can_view_stats_siae_etp
- Un second cas ou tous les membres de la structure peuvent voir les stats etp de leur structure et donc là on tombe bien sur le second cas de figure que je rajoute ici

Reply:
Je serais plutôt partisan de réutiliser `stats_siae_etp` plutôt que de tout dupliquer :
- C'est le même TB
- On filtre sur la même chose : `ASP_SIAE_FILTER_KEY_FLAVOR3`
- La différence n'est que sur les valeurs envoyées dans le filtre, pour `STATS_SIAE_PK_WHITELIST`on ne met que la structure courante, pour `STATS_SIAE_USER_PK_WHITELIST` on met toutes les structures dont la personne est admin.
- Avoir plusieurs trucs qui font quasiment la même chose c'est le meilleur moyen de rater un truc quand on voudra changer les règles d'accès, en plus de nous rajouter du boulot le jour où on voudra complètement ouvrir ou fermer l'accès a ces TB.

Reply:
J'ai corrigé, ce sont deux TBs différents :+1: